### PR TITLE
[Aftershock] Uplifted monsters and zombies

### PR DESCRIPTION
**Pull request by Maleclypse**
#### Summary
SUMMARY: Mods "Uplifted Monsters and zeds"

#### Purpose of change
Continuing to develop Aftershock as a sci-fi environment.

#### Describe the solution
Adds uplifted animals as monsters and their zombified counterparts.

#### Describe alternatives you've considered
Never!

#### Testing
TBD

#### Additional context
Suggestions welcome!  Also the demihuman harvest list will need to be added to #38313 and merged prior to this being merged.

### CODE REVIEW

**Comment by Xenomorph-III** [comment]
I wonder if there are going to be NPC uplifts.

**Comment by Maleclypse** [comment]
That’s part of the Mutant NPCs mod being merged into Aftershock.  Currently there are updated background stories describing various mutants as Uplifts but I have yet to merge the uplift trait and tie it specifically to backgrounds that match up in that PR.

File: data/mods/Aftershock/mobs/uplifted_monsters.json [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+    ],
+    "vision_day": 30,
+    "vision_night": 10,
+    "path_settings": { "max_dist": 10 },
+    "anger_triggers": [ "HURT", "PLAYER_NEAR_BABY" ],
+    "fear_triggers": [ "SOUND" ],
+    "death_function": [ "NORMAL" ],
+    "harvest": "demihuman_large_fur",
+    "reproduction": { "baby_monster": "mon_uplifted_bear_cub", "baby_count": 1, "baby_timer": 700 },
+    "//": "Uplifts will likely outlive mankind",
```
Comment (LaVeyanFiend) [inline data/mods/Aftershock/mobs/uplifted_monsters.json]: What did he mean by this 🤔

Comment (Maleclypse) [reply data/mods/Aftershock/mobs/uplifted_monsters.json]: When I copy pasted from bear there was a // note about how long bears pregnancy lasts.  It was late at night and I decided to keep in the reproduction and then replaced the note with a joke comment.  Now I'm thinking I might keep it as a reminder to add some reproduction to some of the further uplift monsters or to even add locations for uplifted communities of npcs.

Comment (Xenomorph-III) [reply data/mods/Aftershock/mobs/uplifted_monsters.json]: Huh. I always thought that the Uplifts will take over after humanity's extinction. It's a thing in some sci-fi works.

**Comment by aoae23** [comment]
Should the uplifted bears and bear zeds be given XL sized clothing?

**Comment by Maleclypse** [comment]
> Should the uplifted bears and bear zeds be given XL sized clothing?

Absolutely need to do that.  Thank you for the suggestion.  :)

**Pull request merged**